README.md

```diff
@@ -0,0 +1,53 @@
+# `subra-protocol`
+> project subra-protocol v1.0-SNAPSHOT
+
+
+You'll need at least Java 1.8 ([jre](https://www.java.com/download/)
+/[jdk](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html)) 
+to run `subra-protocol`.
+
+## How to use
+
+In order to use `subra-protocol` clone this repository:
+
+```
+  git clone git@github.com:delta-leonis/subra-protocol.git
+```
+
+## Documentation
+
+The javadoc for the current code on `master` can be found on https://delta-leonis.github.io/subra-protocol/
+
+## Building
+
+Make sure you have `gradle>=v2.10` installed. Run the following to build the application:
+
+```
+  gradle build
```

Comment:
So this command creates the jar / compiles the protobuf spul into a jar?

Reply:
this command builds the proto files and injects the generated java into the classpath

Reply:
`gradle build` already deploys libs in `build/lib`